Week-10/url-shortener/src/components/Card.js

```diff
@@ -0,0 +1,10 @@
+const Card = (props) => {
+  return (
+    <div className="card">
+      <i className={props.icon}></i>
+      <p>{props.title}</p>
+    </div>
+  );
+};
```

Comment:
use object destructure
```suggestion
const Card = ({icon, title}) => {
  return (
    <div className="card">
      <i className={icon}></i>
      <p>{title}</p>
    </div>
  );
};
```